# -*- coding: utf-8 -*-
# Generated by the protocol buffer compiler.  DO NOT EDIT!
# NO CHECKED-IN PROTOBUF GENCODE
# source: cadence/api/v1/error.proto
# Protobuf Python Version: 5.29.1
"""Generated protocol buffer code."""
from google.protobuf import descriptor as _descriptor
from google.protobuf import descriptor_pool as _descriptor_pool
from google.protobuf import runtime_version as _runtime_version
from google.protobuf import symbol_database as _symbol_database
from google.protobuf.internal import builder as _builder
_runtime_version.ValidateProtobufRuntimeVersion(
    _runtime_version.Domain.PUBLIC,
    5,
    29,
    1,
    '',
    'cadence/api/v1/error.proto'
)
# @@protoc_insertion_point(imports)

_sym_db = _symbol_database.Default()




DESCRIPTOR = _descriptor_pool.Default().AddSerializedFile(b'\n\x1a\x63\x61\x64\x65nce/api/v1/error.proto\x12\x13uber.cadence.api.v1\"P\n$WorkflowExecutionAlreadyStartedError\x12\x18\n\x10start_request_id\x18\x01 \x01(\t\x12\x0e\n\x06run_id\x18\x02 \x01(\t\"`\n\x14\x45ntityNotExistsError\x12\x17\n\x0f\x63urrent_cluster\x18\x01 \x01(\t\x12\x16\n\x0e\x61\x63tive_cluster\x18\x02 \x01(\t\x12\x17\n\x0f\x61\x63tive_clusters\x18\x03 \x03(\t\"(\n&WorkflowExecutionAlreadyCompletedError\"p\n\x14\x44omainNotActiveError\x12\x0e\n\x06\x64omain\x18\x01 \x01(\t\x12\x17\n\x0f\x63urrent_cluster\x18\x02 \x01(\t\x12\x16\n\x0e\x61\x63tive_cluster\x18\x03 \x01(\t\x12\x17\n\x0f\x61\x63tive_clusters\x18\x04 \x03(\t\"j\n\x1e\x43lientVersionNotSupportedError\x12\x17\n\x0f\x66\x65\x61ture_version\x18\x01 \x01(\t\x12\x13\n\x0b\x63lient_impl\x18\x02 \x01(\t\x12\x1a\n\x12supported_versions\x18\x03 \x01(\t\".\n\x16\x46\x65\x61tureNotEnabledError\x12\x14\n\x0c\x66\x65\x61ture_flag\x18\x01 \x01(\t\"#\n!CancellationAlreadyRequestedError\"\x1a\n\x18\x44omainAlreadyExistsError\"\x14\n\x12LimitExceededError\"\x12\n\x10QueryFailedError\"\"\n\x10ServiceBusyError\x12\x0e\n\x06reason\x18\x01 \x01(\t\"\x1e\n\x1cStickyWorkerUnavailableErrorBZ\n\x17\x63om.uber.cadence.api.v1B\nErrorProtoP\x01Z1github.com/uber/cadence-idl/go/proto/api/v1;apiv1b\x06proto3')

_globals = globals()
_builder.BuildMessageAndEnumDescriptors(DESCRIPTOR, _globals)
_builder.BuildTopDescriptorsAndMessages(DESCRIPTOR, 'cadence.api.v1.error_pb2', _globals)
if not _descriptor._USE_C_DESCRIPTORS:
  _globals['DESCRIPTOR']._loaded_options = None
  _globals['DESCRIPTOR']._serialized_options = b'\n\027com.uber.cadence.api.v1B\nErrorProtoP\001Z1github.com/uber/cadence-idl/go/proto/api/v1;apiv1'
  _globals['_WORKFLOWEXECUTIONALREADYSTARTEDERROR']._serialized_start=51
  _globals['_WORKFLOWEXECUTIONALREADYSTARTEDERROR']._serialized_end=131
  _globals['_ENTITYNOTEXISTSERROR']._serialized_start=133
  _globals['_ENTITYNOTEXISTSERROR']._serialized_end=229
  _globals['_WORKFLOWEXECUTIONALREADYCOMPLETEDERROR']._serialized_start=231
  _globals['_WORKFLOWEXECUTIONALREADYCOMPLETEDERROR']._serialized_end=271
  _globals['_DOMAINNOTACTIVEERROR']._serialized_start=273
  _globals['_DOMAINNOTACTIVEERROR']._serialized_end=385
  _globals['_CLIENTVERSIONNOTSUPPORTEDERROR']._serialized_start=387
  _globals['_CLIENTVERSIONNOTSUPPORTEDERROR']._serialized_end=493
  _globals['_FEATURENOTENABLEDERROR']._serialized_start=495
  _globals['_FEATURENOTENABLEDERROR']._serialized_end=541
  _globals['_CANCELLATIONALREADYREQUESTEDERROR']._serialized_start=543
  _globals['_CANCELLATIONALREADYREQUESTEDERROR']._serialized_end=578
  _globals['_DOMAINALREADYEXISTSERROR']._serialized_start=580
  _globals['_DOMAINALREADYEXISTSERROR']._serialized_end=606
  _globals['_LIMITEXCEEDEDERROR']._serialized_start=608
  _globals['_LIMITEXCEEDEDERROR']._serialized_end=628
  _globals['_QUERYFAILEDERROR']._serialized_start=630
  _globals['_QUERYFAILEDERROR']._serialized_end=648
  _globals['_SERVICEBUSYERROR']._serialized_start=650
  _globals['_SERVICEBUSYERROR']._serialized_end=684
  _globals['_STICKYWORKERUNAVAILABLEERROR']._serialized_start=686
  _globals['_STICKYWORKERUNAVAILABLEERROR']._serialized_end=716
# @@protoc_insertion_point(module_scope)
